src/utils/currency.ts

```diff
@@ -13,8 +13,18 @@ export function roundTwoDecimals(input: string): string {
     return number.round(2).toString();
 }
 
+export function roundUpBigToNearestInteger(x: Big): Big {
+    return x.round(0, 3);
```

Comment:
Can you please add a code comment to the round spec of big: https://mikemcl.github.io/big.js/#round